driver/src/contracts/stablex_auction_element.rs

```diff
@@ -184,7 +184,7 @@ pub mod tests {
         ];
         let mut order_count = HashMap::new();
         StableXAuctionElement::from_bytes(&mut order_count, &bytes);
-        let mut bytes_modified = bytes.clone();
+        let mut bytes_modified = bytes;
```

Comment:
interesting... did clippy point this out?

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FNEvPzZ8bd1V4Y%2Fgiphy-downsized-medium.gif&token=yes" data-gitmeme-token="yes"><img src="https://media2.giphy.com/media/NEvPzZ8bd1V4Y/giphy-downsized-medium.gif" title="Created by gitme.me with /yes"/></a>

Reply:
@fleupold your memes are on point...